clang/include/clang/CIR/MissingFeatures.h

```diff
@@ -230,6 +230,8 @@ struct MissingFeatures {
   static bool shouldInstrumentFunction() { return false; }
   static bool xray() { return false; }
   static bool buildConstrainedFPCall() { return false; }
+  static bool renderScriptTarget() { return false; }
```

Comment:
Looks render script just got killed https://github.com/llvm/llvm-project/commit/af7c58b7ea853ef34462ce97739203e2da3c5894, you can actually remove this, sorry for the extra trouble.